.gitignore

```diff
@@ -17,6 +17,7 @@
 **/Cargo.lock
 **/icicle/build/
 **/wrappers/rust/icicle-cuda-runtime/src/bindings.rs
-**/build/*
+**/build*/*
```

Comment:
This only matches one level deep. Not sure if you meant it to match everything below build folders